### DISCUSSION TITLE
de/blog/2024/03/03/lernos-ki-mooc-vom-65---21062024/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos#15] [General]
# de/blog/2024/03/03/lernos-ki-mooc-vom-65---21062024/

https://lernos.org/de/blog/2024/03/03/lernos-ki-mooc-vom-65---21062024/

**Comment by simondueckert** [comment]
Wir werden auf jeden Fall die Cogneon Akademie in Nürnberg als Lernort anbieten. Wahrscheinlich wird es die Möglichkeit geben, sich dort immer Freitags zum offenen Coworking und Austausch zu treffen. Da ich in den MOOC-Wochen recht eingespannt sein werde, wäre es schön, wenn die Orga des Lernorts und der Termine jemand aus der Region übernehmen könnte. Hat da wer Lust und Zeit zu?

**Comment by KatjaKunzi** [comment]
Eine weitere Lerngruppe wird sich bei der Schweizerischen Post in Bern treffen. Wann, wie und wo genau ist noch in Planung.
